search-index/text2embeddings.py

```diff
@@ -359,7 +358,7 @@ def run_cli(
     description_ids_output_path = (
         output_dir / f"description_ids_{model_name}_{curr_time}.csv"
     )
-    description_data_to_encode["prototype_filename_stem"].to_csv(
+    description_data_to_encode["md5_sum"].to_csv(
```

Comment:
altered this from prototype filename so that the downstream indexing process doesn't break (the loop is now a hash instead of document id).